### DISCUSSION TITLE
A couple suggestions…

### DISCUSSION THREAD

**Discussion by rabbitwithworms** [keiwando/toonsquid#333] [Ideas]
Hello! I’ve got a couple suggestions regarding your pixel engine. This is my first time suggesting things so I hope I don’t come off as rude.

### Suggestion 1: Something to fix this brush with?

So I have this really cool pixel brush that works using a pattern with the Subtract blend mode, made by setting Contrast to 100% and Brightness to 40%. Unfortunately it does not seem to work well in ToonSquid…

Here is what the dithering brush normally looks like in Artstudio Pro:

![IMG_4255](https://github.com/keiwando/toonsquid/assets/132721101/9c8ef569-84b8-4b10-94aa-6f3807a69faf)

Here is roughly the same brush using ToonSquid:

![IMG_4254](https://github.com/keiwando/toonsquid/assets/132721101/c8da909a-75f0-424a-bbe3-ea4f2e1156b6)

Artstudio is based on Photoshop, so maybe how the brush engine is set up might be a problem? Or is it a bug? I don’t really know for sure. Maybe it’s the engine.

### Suggestion 2: Pattern Bucket Tool

I’d be nice to have a pattern bucket tool, it’s self explanatory. You pick a pattern and it fills the entire page with it.

### Suggestion 3:

Adjustment layers! Like Posterize, Grayscale, yknow the ones. They’re filters except they’re on layers, and you can do all sorts of stuff with them. I cheated making a cool pixel arty sunset by using a stock photo on pixabay, said pattern bucket tool and using these layers:

![IMG_4249](https://github.com/keiwando/toonsquid/assets/132721101/a501cb18-aa26-46bd-a959-07478d638a1f)

That’s it pretty much, ToonSquid is a snazzy animation program and these changes could make it even better for bitmap/pixel artists like me.

**Comment by VeiloPot** [comment]
> Hello! I’ve got a couple suggestions regarding your pixel engine. This is my first time suggesting things so I hope I don’t come off as rude.
> 
> 
> 
> ### Suggestion 1: Something to fix this brush with?
> 
> 
> 
> So I have this really cool pixel brush that works using a pattern with the Subtract blend mode, made by setting Contrast to 100% and Brightness to 40%. Unfortunately it does not seem to work well in ToonSquid…
> 
> 
> 
> Here is what the dithering brush normally looks like in Artstudio Pro:
> 
> 
> 
> ![IMG_4255](https://github.com/keiwando/toonsquid/assets/132721101/9c8ef569-84b8-4b10-94aa-6f3807a69faf)
> 
> 
> 
> Here is roughly the same brush using ToonSquid:
> 
> 
> 
> ![IMG_4254](https://github.com/keiwando/toonsquid/assets/132721101/c8da909a-75f0-424a-bbe3-ea4f2e1156b6)
> 
> 
> 
> Artstudio is based on Photoshop, so maybe how the brush engine is set up might be a problem? Or is it a bug? I don’t really know for sure. Maybe it’s the engine.
> 
> 
> 
> ### Suggestion 2: Pattern Bucket Tool
> 
> 
> 
> I’d be nice to have a pattern bucket tool, it’s self explanatory. You pick a pattern and it fills the entire page with it.
> 
> 
> 
> ### Suggestion 3:
> 
> 
> 
> Adjustment layers! Like Posterize, Grayscale, yknow the ones. They’re filters except they’re on layers, and you can do all sorts of stuff with them. I cheated making a cool pixel arty sunset by using a stock photo on pixabay, said pattern bucket tool and using these layers:
> 
> 
> 
> ![IMG_4249](https://github.com/keiwando/toonsquid/assets/132721101/a501cb18-aa26-46bd-a959-07478d638a1f)
> 
> 
> 
> 
> 
> That’s it pretty much, ToonSquid is a snazzy animation program and these changes could make it even better for bitmap/pixel artists like me.
> 
> 
> 
> 

Omg I actually would like to see a posterize feature, it might come in handy!

**Comment by keiwando** [comment]
Thank you very much for the suggestions!
